index.html

```diff
@@ -29,9 +29,11 @@
             <a href="" class="font_2">PROJECTS</a>
             <a href="" class="font_2">CONTACT</a>
 
+            <button class="nav__button--callnow"><a href="" class="font_2">CALL NOW</a>
```

Comment:
w BEM (sposb pisania klas) `--text` text powinien wskazywac modyfikację, odstępstwo od część wcześniejszej np nav -- to nawigacja, nav__button - to wskazuje na buttony w nawigacjach, nav__button--fsize18 -> tak np bym opisał gdyby np 1 button sposrod reszty mial byc font size 18

Reply:
chyba rozumiem - poprawiam